script/SearchResults.js

```diff
@@ -0,0 +1,119 @@
+class Stock {
+
+    constructor(searchPhrase) {
+        this.searchQuery = searchPhrase
+        this.queryUrl = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/search?query=${this.searchQuery}&limit=10&exchange=NASDAQ`;
+        this.queryResult = this.getStocks()       
+    };
+
+    setIsLoading(isLoading) {
+
+        const spinner = document.getElementById('spinner').style;
+        if (isLoading) {
+            return  spinner.display = "block" ;
+        } else spinner.display = "none";
```

Comment:
For this you can use ternary operator like so:
spinner.display = isLoading? "block" : "none" ;